.clang-format

```diff
@@ -1,118 +0,0 @@
-# SPDX-License-Identifier: GPL-2.0
```

Comment:
Are we certain that nobody's actually using this? I'd hate to remove it had have to add it back again immediately because someone was using it.

Reply:
We are all using this - why is it being removed?